packages/api-server/src/server.ts

```diff
@@ -6,15 +6,15 @@ export interface HttpServerParams {
   fastify: FastifyInstance
 }
 
-export const startServer = ({
+export const startServer = async ({
   port = 8911,
   socket,
   fastify,
 }: HttpServerParams) => {
   const host = process.env.NODE_ENV === 'production' ? '0.0.0.0' : '::'
   const serverPort = socket ? parseInt(socket) : port
 
-  fastify.listen({ port: serverPort, host })
+  await fastify.listen({ port: serverPort, host })
```

Comment:
We need to await this so that `fastify.listeningOrigin` isn't null when we go to use it